VideoWeb/VideoWeb.UnitTests/Controllers/UserDataControllerTest.cs

```diff
@@ -20,24 +22,25 @@ public class UserDataControllerTest
     {
         private AutoMock _mocker;
         private UserDataController _sut;
+        private VhoConferenceFilterQuery _query;
 
         [SetUp]
         public void Setup()
         {
             _mocker = AutoMock.GetLoose();
-            _mocker.Mock<IMapperFactory>().Setup(x => x.Get<IEnumerable<UserResponse>, IEnumerable<string>, List<CourtRoomsAccountResponse>>()).Returns(_mocker.Create<CourtRoomsAccountResponseMapper>());
+            _mocker.Mock<IMapperFactory>().Setup(x => x.Get<IEnumerable<ConferenceForAdminResponse>, List<CourtRoomsAccountResponse>>()).Returns(_mocker.Create<CourtRoomsAccountResponseMapper>());
             _sut = _mocker.Create<UserDataController>();
+            _query = new VhoConferenceFilterQuery { HearingVenueNames = new List<string> { "Venue Name 01", "Venue Name 02" } };
         }
 
         [Test]
         public async Task Should_return_list_of_court_rooms_accounts_with_status_ok()
         {
-            var accounts = UserResponseBuilder.BuildData();
-            var usernames = new VhoConferenceFilterQuery { UserNames = new List<string> { "Manual01", "Manual03" } };
+            var conferences = ConferenceResponseBuilder.BuildData();
 
-            _mocker.Mock<IUserApiClient>().Setup(x => x.GetJudgesAsync()).ReturnsAsync(accounts);
+            _mocker.Mock<IVideoApiClient>().Setup(x => x.GetConferencesTodayForAdminByHearingVenueNameAsync(null, _query.HearingVenueNames)).ReturnsAsync(conferences);
 
-            var result = await _sut.GetCourtRoomsAccounts(usernames);
+            var result = await _sut.GetCourtRoomsAccounts(_query);
 
             var typedResult = (OkObjectResult)result.Result;
             typedResult.Should().NotBeNull();
```

Comment:
Can you rename judgeList to something more fitting like courtRoomAccounts and maybe change like "FirstName1" to "Venue1" and "LastName1" to "Room1". Sorry to be picky but it makes the test hard to read.